### DISCUSSION TITLE
STeamVR Disconnected

### DISCUSSION THREAD

**Discussion by whitfieldts** [alvr-org/ALVR#1953] [General]
I am trying to get ALVR working in NixOS.  I downloaded the .tar.gz and was able to get that to run by setting the interpreter.  When I click "Launch SteamVR" it launches steamVR correctly also.  The problem becomes that once SteamVR is launched, ALVR still says SteamVR: Disconnected

I was wondering if anyone could tell me what might cause this.  I tried opening firewall ports on this system, but the connection issue seems be on the same computer.  My understanding is that I should not expect the headset to be able to connect until after the streamer connection works.

**Comment by kostek001** [comment]
ALVR AppImage works on NixOS when run with `appimage-run`.

**Reply by whitfieldts** [reply]
That worked! Thanks I had never heard of appimage-run before.
